packages/dropdowns/src/styled/items/media/StyledMediaFigure.ts

```diff
@@ -30,7 +33,9 @@ export const StyledMediaFigure = styled(
   'data-garden-version': PACKAGE_VERSION
 })<IStyledMediaFigureProps>`
   float: ${props => (props.theme.rtl ? 'right' : 'left')};
-  margin-top: ${props => props.theme.space.base * 0.5}px;
+  /* stylelint-disable declaration-no-important */
+  margin-top: ${props => props.theme.space.base * 0.5}px !important; /* [1] */
```

Comment:
I'm curious, where is there a `margin-top` in StyledAvatar? I'm not seeing it.

Reply:
The `margin: 0; /* <figure> reset */` is what is winning right now

Reply:
I keep thinking `margin-top` beats out `margin` in terms of specificity. But, alas, this is not true.